formats/frontend_links_definition.json

```diff
@@ -47,11 +60,7 @@
       "links": {
         "type": "object",
         "additionalProperties": true,
```

Comment:
We can use [`patternProperties`](https://spacetelescope.github.io/understanding-json-schema/reference/object.html#pattern-properties) here. Something like:

```json
       "links": {
          "type": "object",
          "patternProperties": {
            "^[a-z_]+$": {
              "$ref": "#/definitions/frontend_links"
            }
          }
        }
```

This allows us to validate that what is coming from content store has the basic required fields, but that object will also have `additionalProperties: true`